Cargo.toml

```diff
@@ -21,25 +21,19 @@ incremental = true
 lto = "fat"
 opt-level = 3
 panic = "unwind"
-strip = true
-
-# For testing cargo bloat
-[profile.release-no-strip]
-inherits = "release"
-strip = false
 
 # This one optimizes binary size
 [profile.release-small]
 inherits = "release"
 opt-level = "z"
+strip = true
 
 [profile.release-big]
 inherits = "release"
 codegen-units = 16
 incremental = true
 lto = "thin"
 opt-level = 1
-panic = "unwind"
```

Comment:
not sure what this change does

Reply:
Default behaviour I belive, but I moved it to the `release` profile, since both `small` and `big` specified it, might as well have them inherit it.